### PR TITLE
i.sentinel.mask: improved control checks about the number of clouds and shadows detected

### PR DESCRIPTION
**Pull request by RobiFag**
I have modified the code in order to manage the cases in which no clouds or shadows are detected. In this way, the issue #85 should be solved.

### CODE REVIEW

File: grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py [hunk 1]
```diff
@@ -99,15 +99,15 @@
 #%option
 #% key: cloud_threshold
 #% type: integer
-#% description: threshold for cleaning small areas from cloud mask
+#% description: threshold (sm) for cleaning small areas from cloud mask
```
Comment (veroandreo) [inline grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py]:
```suggestion
#% description: threshold for cleaning small areas from cloud mask (in square meters)
```

File: grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py [hunk 2]
```diff
@@ -99,15 +99,15 @@
...
 #% required : yes
 #% answer: 50000
 #% guisection: Output
 #%end
 #%option
 #% key: shadow_threshold
 #% type: integer
-#% description: threshold for cleaning small areas from shadow mask
+#% description: threshold (sm) for cleaning small areas from shadow mask
```
Comment (veroandreo) [inline grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py]:
```suggestion
#% description: threshold for cleaning small areas from shadow mask (in square meters)
```

**Comment by veroandreo** [comment]
@RobiFag would you merge this so it is easier to test? thanks a lot! :)

**Comment by neteler** [comment]
I take liberty to merge the changes for easier testing.

**Comment by RobiFag** [comment]
@veroandreo sorry!! I completely forgot to merge the PR!! 
Thanks @neteler

**Pull request merged**